react/src/lib/Popover/index.js

```diff
@@ -15,6 +16,25 @@ class Popover extends React.Component {
     this.props.startOpen && this.forceUpdate();
   }
 
+  componentDidUpdate(prevProps, prevState) {
+    // focus on the first button in the EventOverlay
+    const { isOpen } = this.state;
+    const { autoFocusOnFirstInteractiveElt } = this.props;
```

Comment:
I think in other places this same prop is only called `autoFocus`. Can you check? It would be nice to keep it consistent

Reply:
I'll double check

Reply:
I can change it to autoFocus makes sense to me